components/AvaragePriceGraph.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { ComposedChart, Area, YAxis, Tooltip, ResponsiveContainer, XAxis } from 'recharts';
+import { formatBidValue } from '../utils/utils';
+
+export default function AvaragePriceGraph({
+  periodData,
+  backgroundShadowLimit,
+}: {
+  periodData:
+    | {
+        id: number;
+        price: string | null;
+        startTime: number;
+      }[]
+    | undefined;
```

Comment:
use `?` shortcut instead of `| undefined`
```suggestion
  periodData?:
    | {
        id: number;
        price: string | null;
        startTime: number;
      }[];
```